README.md

```diff
@@ -587,6 +587,64 @@ list slicing notation:
 new_table = old_table[0:5]
 ```
 
+## Themes
+
+### Getting Themes
+
+If you want your tables to be even fancier (if that's possible) then you can use color
+themes. All you have to do is import it.
+
+```python
+from prettytable import PrettyTable, THEME
+```
+
+> Note, if you are using Windows, type `pip install prettytable[windows]`, so the color
+> codes get reformatted to your OS. If color does not show for other operating systems,
+> you can still do this and that may fix your problem.
+
+### Using Themes
+
+After you are certain that you've imported the theme, you can quickly apply it. Now,
+printing the theme will include color.
+
+```python
+x = PrettyTable()
+x.set_theme(THEME.OCEAN)
+print(x)
+```
+
+You can also set theme on import.
+
+```python
+x = PrettyTable(theme = THEME.OCEAN)
+print(x)
+```
+
+### Custom Themes
+
+If you don't want any of the themes supplied, you can make your own. Create a custom
+dictionary with the following keys:
+
+```python
+my_theme = {
+  "base": "\033[31m",
+  "border": "\033[33m",
+  "decor": "\033[0m"
```

Comment:
The code uses "junction" for "decor", and it's also exposed via the API, so let's replace all "decor" with junction".

```suggestion
  "junction": "\033[0m"
```